### PR TITLE
Pjs annotations

### PR DESCRIPTION
**Pull request by dekiel**
**Description**

Changes proposed in this pull request:

- Added annotations to all prowjobs identified in excel sheet.
- Fixed trigger for pjconfigtests prowjob to run on correct path in test-infra repo.

### CODE REVIEW

**Review by Disper — APPROVED** [review]
I don't see any framefrog team jobs so, lgtm

File: prow/jobs/busola/integration-k3s-test.yaml [hunk 1]
```diff
@@ -5,6 +5,8 @@ presubmits: # runs on PRs
   kyma-project/busola:
     - name: pre-busola-integration-test-k3s-cluster
       annotations:
+        description: "busola testing"
```
Comment (mrCherry97) [inline prow/jobs/busola/integration-k3s-test.yaml:8]:
```suggestion
        description: "Busola integration testing"
```

File: prow/jobs/busola/integration-k3s-test.yaml [hunk 2]
```diff
@@ -52,6 +54,8 @@ presubmits: # runs on PRs
             dedicated: "high-cpu"
     - name: pre-busola-integration-test-k3s-namespace
       annotations:
+        description: "busola testing"
```
Comment (mrCherry97) [inline prow/jobs/busola/integration-k3s-test.yaml:57]:
```suggestion
        description: "Busola integration testing"
```

File: prow/jobs/busola/integration-k3s-test.yaml [hunk 3]
```diff
@@ -102,6 +106,8 @@ postsubmits: # runs on main
   kyma-project/busola:
     - name: post-busola-integration-test-k3s-cluster
       annotations:
+        description: "busola testing"
```
Comment (mrCherry97) [inline prow/jobs/busola/integration-k3s-test.yaml:109]:
```suggestion
        description: "Busola integration testing"
```

File: prow/jobs/busola/integration-k3s-test.yaml [hunk 4]
```diff
@@ -150,6 +156,8 @@ postsubmits: # runs on main
             dedicated: "high-cpu"
     - name: post-busola-integration-test-k3s-namespace
       annotations:
+        description: "busola testing"
```
Comment (mrCherry97) [inline prow/jobs/busola/integration-k3s-test.yaml:159]:
```suggestion
        description: "Busola integration testing"
```

**Review by marcobebway — APPROVED** [review]

**Review by mrCherry97 — CHANGES_REQUESTED** [review]

**Review by Disper — DISMISSED** [review]

File: prow/jobs/control-plane/components/schema-migrator/schema-migrator-kcp-generic.yaml [hunk 5]
```diff
@@ -5,7 +5,7 @@ presubmits: # runs on PRs
   kyma-project/control-plane:
     - name: pre-control-plane-components-schema-migrator
       annotations:
-        owner: "gophers, framefrog"
```
Comment (ralikio) [inline prow/jobs/control-plane/components/schema-migrator/schema-migrator-kcp-generic.yaml:8]: In some places there is no space but in others there is. Is that intentional?

File: prow/jobs/control-plane/tests/e2e/provisioning/e2e-provisioning-test-build.yaml [hunk 6]
```diff
@@ -6,7 +6,7 @@ presubmits: # runs on PRs
     - name: pull-e2e-provisioning-test-build
       annotations:
         description: "E2E provisioning test build on PR"
```
Comment (ralikio) [inline prow/jobs/control-plane/tests/e2e/provisioning/e2e-provisioning-test-build.yaml:8]: I am not sure if we should be responsible for that test. Let me check.

File: prow/jobs/control-plane/components/schema-migrator/schema-migrator-kcp-generic.yaml (hunk 5 above)
Comment (Ressetkk) [reply prow/jobs/control-plane/components/schema-migrator/schema-migrator-kcp-generic.yaml:8]: Nope, but it shouldn't matter anyway.

**Review by ralikio — CHANGES_REQUESTED** [review]
One comment about "spaces" left above.

File: prow/jobs/control-plane/components/schema-migrator/schema-migrator-kcp-generic.yaml (hunk 5 above)
Comment (ralikio) [reply prow/jobs/control-plane/components/schema-migrator/schema-migrator-kcp-generic.yaml:8]: Could you please remove the space? Thank you in advance.

**Review by ralikio — APPROVED** [review]

**Review by mrCherry97 — APPROVED** [review]

**Review by Disper — APPROVED** [review]

**Review by friedrichwilken — APPROVED** [review]

**Review by kwiatekus — APPROVED** [review]

**Review by dbadura — APPROVED** [review]

**Review by ralikio — APPROVED** [review]

**Review by Tomasz-Smelcerz-SAP — APPROVED** [review]

**Review by Ressetkk — APPROVED** [review]

**Pull request merged**